src/datatypes/dataset.ts

```diff
@@ -193,25 +194,28 @@ export interface Formatter extends Lookup<Function | undefined> {
   'DATASET'?: (v: Dataset) => string;
 }
 
-let defaultFormatter: Formatter = {
-  'NULL': (v: any) => 'NULL',
-  'TIME': (v: Date) => v.toISOString(),
-  'TIME_RANGE': (v: TimeRange) => '' + v,
-  'SET/TIME': (v: Set) => '' + v,
-  'SET/TIME_RANGE': (v: Set) => '' + v,
-  'STRING': (v: string) => '' + v,
-  'SET/STRING': (v: Set) => '' + v,
-  'BOOLEAN': (v: boolean) => '' + v,
-  'NUMBER': (v: number) => '' + v,
-  'NUMBER_RANGE': (v: NumberRange) => '' + v,
-  'SET/NUMBER': (v: Set) => '' + v,
-  'SET/NUMBER_RANGE': (v: Set) => '' + v,
-  'DATASET': (v: Dataset) => 'DATASET'
-};
+function getDefaultFormatter(tz: Timezone = Timezone.UTC): Formatter {
+  return {
+    'NULL': (v: any) => 'NULL',
+    'TIME': (v: Date) => moment.tz(v, tz.toString()).format(),
+    'TIME_RANGE': (v: TimeRange) => '' + v,
```

Comment:
I think `TIME_RANGE` should also use the timezone and pass it into `TimeRange#toString()` same with `SET/TIME*`